apps/gateway/src/routes/api/v1/chats/handlers/addMessage.ts

```diff
@@ -22,17 +23,26 @@ export const addMessageHandler = factory.createHandlers(
     return streamSSE(
       c,
       async (stream) => {
-        const { documentLogUuid, messages, source } = c.req.valid('json')
+        const { uuid, messages } = c.req.valid('json')
         const workspace = c.get('workspace')
 
         const result = await addMessages({
           workspace,
-          documentLogUuid,
+          documentLogUuid: uuid,
           messages,
-          source,
+          source: LogSources.API,
```

Comment:
I know this is not technically correct because we use ts sdk in playground but honestly i don't care